frontend/components/MainPage/sections/AlertSections/LowFunds/LowOperatingBalanceAlert.tsx

```diff
@@ -13,29 +12,30 @@ import { useLowFundsDetails } from './useLowFunds';
 
 const { Text, Title } = Typography;
 
+/**
+ * Alert for low operating (safe) balance
+ */
 export const LowOperatingBalanceAlert = () => {
   const { storeState } = useStore();
-  const { selectedAgentConfig, selectedAgentType } = useServices();
+  const { selectedAgentType } = useServices();
   const { isLoaded: isBalanceLoaded, masterSafeNativeGasBalance } =
     useMasterBalances();
 
-  const { chainName, tokenSymbol, masterSafeAddress } = useLowFundsDetails();
+  const { chainName, tokenSymbol, masterSafeAddress, masterThresholds } =
+    useLowFundsDetails();
 
   const isLowBalance = useMemo(() => {
     if (!masterSafeNativeGasBalance) return false;
+    if (!masterThresholds) return false;
+
     return (
       masterSafeNativeGasBalance <
-      selectedAgentConfig.operatingThresholds[WalletOwnerType.Master][
-        WalletType.Safe
-      ][CHAIN_CONFIG[selectedAgentConfig.evmHomeChainId].nativeToken.symbol]
+      masterThresholds[WalletType.Safe][tokenSymbol]
```

Comment:
since funds are transferred from master safe to agents safes, should we check agent safes balances here as well? master safe can have very low balance, when agents are still operational 

this is the old code from staging
<img width="621" alt="image" src="https://github.com/user-attachments/assets/970a6d65-889c-4f51-88a7-f85db54b2fc8" />

Reply:
If the Agent safe has funds to operate but the Master safe has less than 2XDAI the message should appear but not block the start agent button.